tests/languages/docker_test.py

```diff
@@ -3,6 +3,188 @@
 from pre_commit.languages import docker
 from pre_commit.util import CalledProcessError
 
+DOCKER_NOT_ROOTLESS_SYSTEM_INFO = '''
+Client:
+ Debug Mode: false
+
+Server:
+ Containers: 8
+  Running: 0
+  Paused: 0
+  Stopped: 8
+ Images: 2
+ Server Version: 19.03.8
+ Storage Driver: overlay2
+  Backing Filesystem: <unknown>
+  Supports d_type: true
+  Native Overlay Diff: true
+ Logging Driver: journald
+ Cgroup Driver: systemd
+ Plugins:
+  Volume: local
+  Network: bridge host ipvlan macvlan null overlay
+  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
+ Swarm: inactive
+ Runtimes: runc
+ Default Runtime: runc
+ Init Binary: /usr/libexec/docker/docker-init
+ containerd version:
+ runc version: fbdbaf85ecbc0e077f336c03062710435607dbf1
+ init version:
+ Security Options:
+  seccomp
+   Profile: default
+  selinux
+ Kernel Version: 5.6.14-300.fc32.x86_64
+ Operating System: Fedora 32 (Thirty Two)
+ OSType: linux
+ Architecture: x86_64
+ CPUs: 8
+ Total Memory: 15.29GiB
+ Name: carbon
+ ID: IUGE:4L5B:VVTV:JGIA:IIJN:G7MG:TVGF:XBVW:YHF7:MYRK:L524:4HBK
+ Docker Root Dir: /var/lib/docker
+ Debug Mode: false
+ Registry: https://index.docker.io/v1/
+ Labels:
+ Experimental: false
+ Insecure Registries:
+  127.0.0.0/8
+ Live Restore Enabled: true
+'''  # noqa
```

Comment:
it's usually better to specify specific violation codes instead of ignoring everything